packages/@aws-cdk-testing/cli-integ/lib/shell.ts

```diff
@@ -14,21 +14,19 @@ export async function shell(command: string[], options: ShellOptions = {}): Prom
     throw new Error('Use either env or modEnv but not both');
   }
 
+  const outputs = new Set(options.outputs);
```

Comment:
Set of streams? Is that a good idea? Though I can't think of a reason why someone would want to reuse the same output...